src/containers/tutor-home-page/add-photo-step/AddPhotoStep.jsx

```diff
@@ -13,7 +13,7 @@ import { validationData } from './constants'
 
 const AddPhotoStep = ({ btnsBox, stepLabel }) => {
   const { isLaptopAndAbove, isTablet, isMobile } = useBreakpoints()
-  const [photoError, setPhotoError] = useState(null)
+  const [photoError, setPhotoError] = useState()
```

Comment:
what was the purpose to make undefined as the default value instead of null?

Reply:
I did this because `UploadFileEmitter` accepts optional error like following:
```jsx
type UploadFileEmitterArgs = {
  files: File[]
  error?: string
}

export type UploadFileEmitter = (args: UploadFileEmitterArgs) => void
```
I think it will be finally close all inaccuracies after we convert `AddPhotoStep` component and other components related to file uploading to typescript.

Reply:
Speaking about `error ?? ''` inside `AddDocuments.tsx`, I have two options to fix type error with current implementation as shown below:

![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/109922101/3c50f2e0-ecce-42a6-a23b-aa0638bc531e)

Please tell me which one should I pick

I think the same issue will be inside `AddPhotoStep.jsx`, but it is not converted to typescript yet.

Reply:
I prefer to use the second option, but as I see, you changed an error to optional, and that`s why it throws an error, can you make the error not optional in Emmiter?

Reply:
Made `Emitter` error optional